s3_to_redshift.go

```diff
@@ -44,9 +46,20 @@ func fatalIfErr(err error, msg string) {
 	}
 }
 
+// rounds down a dateTime to the timeGranularity
+func truncateDate(date time.Time) time.Time {
```

Comment:
Good points. Yup, the desired behavior is to truncate. Added additional parameter + comments in f69d70e